RNTester/js/RNTesterExampleContainer.js

```diff
@@ -37,9 +38,25 @@ class RNTesterExampleContainer extends React.Component {
       return <this.props.module />;
     }
 
+    const filter = ({example, filterRegex}) => filterRegex.test(example.title);
```

Comment:
What do you think about keeping the old behavior for examples that only export one example?

```suggestion
    if (this.props.module.examples.length === 1) {
      const Example = this.props.module.examples[0].render;
      return <Example />;
    }
    
    const filter = ({example, filterRegex}) => filterRegex.test(example.title);
```